reservation/test_schedule_timings.py

```diff
@@ -0,0 +1,132 @@
+from django.test import TestCase
+from django.urls import reverse
+from django.utils import timezone
+from datetime import timedelta
+from reservation.models import Schedule
+from account.models.professional import Professional, Professions
+from account.models.profile import Profile, UserType
+from reservation.forms import ScheduleForm
+from django.contrib.auth.models import User
+
+current_datetime = timezone.now()
+
+
+class ScheduleTestCase(TestCase):
+    def setUp(self):
+        self.user = User.objects.create_user(
+            username='testuser',
+            password='testpass'
+        )
+        self.profile = Profile.objects.create(
+            user_id=self.user,
+            phone_number='123456789',
+            country='USA',
+            city='New York',
+            address='123 Main St',
+            user_type=UserType.Professional
+        )
+        self.professional = Professional.objects.create(
+            profile_id=self.profile,
+            profession=Professions.Locksmith,
+            description='Test Description')
+
+        self.schedule = Schedule.objects.create(
+            professional_id=self.professional,
+            start_day=(current_datetime + timedelta(days=5)).replace(hour=10, minute=0,
+                                                                     second=0, microsecond=0),
+            end_day=(current_datetime + timedelta(days=5)).replace(hour=18, minute=0,
+                                                                   second=0, microsecond=0),
+            meeting_time=60
+        )
+
+    def test_create_schedule(self):
+        self.client.force_login(self.user)
```

Comment:
Please avoid login for now

Reply:
I can't use without the login because when I want to find the all schedule of a certain professional, I have to know who the professional is who is connected to the system like I did here: schedule = list(Schedule.objects.filter(professional_id__profile_id__user_id=request.user)) Unless you want it to be written there for example professional_id=2
In addition, I don't think it should hurt the rest of the PR and in any case I will have to do it in the future

Reply:
Correct, 
But you implement another feature,
For demo usage and testing.
You can use ClientID and ProffessionalID already entered by fixtures

Reply:
Done